### DISCUSSION TITLE
CARTO3

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#6276] [Ideas] [answered]
Hi all.
CARTO is moving away from CARTO 2 which is supported in Terria.js (https://docs-v8.terria.io/guide/connecting-to-data/catalog-type-details/carto/) to CARTO 3 which generates vector tiles instead of raster tiles (https://api-docs.carto.com/#75feef02-1e8d-4d95-be36-17276228544a). It would be really cool if we can use CARTO 3 layers with Terria.js.

**Comment by nf-s** [comment, marked as answer]
We will be starting Carto3 support soon 🙂 

Investigation ticket here https://github.com/TerriaJS/terriajs/issues/6323
